### ISSUE TITLE
4 Pod faults in 24 hours - Has EROS communication changed?

### ISSUE THREAD

**Issue by Technixx** [nightscout/AndroidAPS#1602]
Hi, 

My son is using AAPS v3
Unihertz Jelly2, Android 10 
I use Orangelink Pro and Original with firmware 3.2. - fresh batteries

I have now had 2 screamers in 24 hours. 
I have problems with activation and pod unreachable 2 pods failing multiple times during activation with fault: 049 | 0x31 | Incorrect pod state for command

I am now at this time 10 April at 23:17 - on the 5th pod - still getting errors but managed to activate this one.

It seems like it can fail multiple times, then get the correct "GET STATUS" for a few minutes then back to pod fault.

It has started with the 3.0.0.1 and the new Lot version for Eros. I can't tell if it's either one of them. 
![Screenshot_20220410-221858](https://user-images.githubusercontent.com/30857763/162640391-381b7b55-d729-446b-a6e3-5097fa9c54f3.jpg)
![Screenshot_20220407-235614](https://user-images.githubusercontent.com/30857763/162640392-e1913d98-ce14-456e-92ef-32811818606d.jpg)
![Screenshot_20220408-000832](https://user-images.githubusercontent.com/30857763/162640394-b570de0b-33ab-4f37-9ecf-004f12a517e0.jpg)
![Screenshot_20220408-000925](https://user-images.githubusercontent.com/30857763/162640395-62bcba96-8443-4e2f-9713-5824777cf484.jpg)
![Screensh
[AndroidAPS.log](https://github.com/nightscout/AndroidAPS/files/8459936/AndroidAPS.log)
ot_20220408-001415](https://user-images.githubusercontent.com/30857763/162640396-b29e9aa4-262e-49d2-b2d4-bae23f9e96f2.jpg)
![Screenshot_20220410-215523](https://user-images.githubusercontent.com/30857763/162640397-e9debee1-1e7f-40e7-8c54-c29f9c0ca5d7.jpg)
![Screenshot_20220410-221436](https://user-images.githubusercontent.com/30857763/162640398-b6bd4e4c-3607-43cd-a0c7-1a52723bc203.jpg)
![Screenshot_20220410-221531](https://user-images.githubusercontent.com/30857763/162640399-867ae741-312b-40be-bf83-520abedabc33.png)
![Screenshot_20220410-221615](https://user-images.githubusercontent.com/30857763/162640401-01c2af0d-3cd6-48e1-8358-600acbdabcbe.jpg)
![Screenshot_20220410-221756](https://user-images.githubusercontent.com/30857763/162640402-ff5169be-ef15-4109-8305-fa9bbed57513.jpg)
![20220408_073131](https://user-images.githubusercontent.com/30857763/162640503-cfbb5d45-5785-477c-8973-6733a69b007b.jpg)
![20220408_073312](https://user-images.githubusercontent.com/30857763/162640504-688a8e2e-f19d-4033-8a20-ed567a5f0438.jpg)

**Comment by ninelore** [comment]
As mentioned on Discord: Verify the batch with the original PDM to rule out a bad batch.

**Comment by Technixx** [comment]
I have tried to use 3 more pods now, without these problems. I have scrolled through multiple logs from, and not found anything unusual. I am still unsure about why this incidents occurred and why they happened in a row. But it might actually have been just bad pods. I know thy try to mix these pods to minimize this risk, but it seems like it might have been a box of multiple bad pods coincidentally. I will reopen the issue if this happens again.
